build/yamls/base/conf/antrea-agent.conf

```diff
@@ -97,7 +97,7 @@ featureGates:
 # If PORT is empty, we default to 4739, the standard IPFIX port.
 # If no PROTO is given, we consider "tcp" as default. We support "tcp" and "udp"
 # L4 transport protocols.
-#flowCollectorAddr: "flow-aggregator.flow-aggregator.svc:tcp"
+#flowCollectorAddr: "flow-aggregator.flow-aggregator.svc:4739:tcp"
```

Comment:
wasn't this supposed to be just "flow-aggregator.flow-aggregator.svc" originally?

Reply:
Not sure if I had that before. Picked that one because we could represent all default values.